auth.php

```diff
@@ -114,7 +114,7 @@ public function user_signup($user, $notify=true) {
         if ($notify) {
             if (!send_confirmation_email($user)) {
                 // TODO make this more resilient? Email shouldn't be critical here.
-                print_error('noemail', 'auth_enrolkey');
+                // print_error('noemail', 'auth_enrolkey');
```

Comment:
ok, will leave print_error there. actually with that line commented the codecheckes complains too on empty IF statement.